charters/wot-wg-2023-draft.html

```diff
@@ -491,6 +491,113 @@ <h3 id="w3c-coordination">W3C Groups</h3>
+                TBD.
+              </dd>
+            </dt>
+            <dt> <a href="https://www.ietf.org/">IETF</a>
+              <dd>
+                TBD.
+              </dd>
+            </dt>
+            <dt> <a href="https://csa-iot.org/">Connectivity Standards Alliance (CSA)</a>
+            <dd>
+              TBD. However, a Matter Protocol Binding would be of interest.
+            </dd>
+            </dt>
+            <dt> <a href="https://onedm.org/">One Data Model</a>
+            <dd>
+              For coordination of SDF with the TD.
+            </dd>
+            </dt>
+            <dt> <a href="https://www.ogc.org/">Open Geospatial Consortium (OGC)</a>
+            <dd>
+              TBD. However, geolocation related collaboration is of interest.
+            </dd>
+            </dt>
+            <dt> <a href="https://www.conexxus.org/">Conexxus</a>
+            <dd>
+              TBD. However, retail use cases exploration is probably what we are doing.
+            </dd>
+            </dt>
+          </dl>
+          <p>
+            List from previous charter to be verified. Add them to the list above if agreed.
+            There is also a wiki page linked but that page has not been updated since 2018.
+          </p>
+          
+          <p>The following list provides examples of organizations with which the WoT Working Group 
+            has already had significant coordination during the period of the first charter. 
+            A <a href="https://www.w3.org/WoT/IG/wiki/How_does_W3C_compare_to_other_organizations">longer list</a> 
+            is available on the Interest Group wiki.
+          </p>
+          <dl>
+            <dt><a href="https://datatracker.ietf.org/doc/charter-irtf-t2trg/">IRTF Thing to Thing Research Group</a></dt>
+            <dd>For coordination of matters of mutual interest in relation to the Web of Things, such as data modelling, discovery, directory services, and IoT semantics.</dd>
+            <dt><a href="https://openconnectivity.org">Open Connectivity Foundation</a></dt>
+            <dd>For collaboration on integrating OCF based platforms within the Web of Things, including platform metadata and approaches for enabling semantic interoperability, and end to end security across platforms.</dd>
+            <dt><a href="https://echonet.jp/english/">ECHONET Consortium</a></dt>
+            <dd>For collaboration on integrating ECHONET Consortium based platforms within the Web of Things, including platform metadata and approaches for enabling semantic interoperability, and end to end security across platforms.</dd>
+            <dt><a href="http://www.iiconsortium.org">Industrial Internet Consortium</a></dt>
+            <dd>For collaboration on testbeds and on gathering user requirements for the Web of Things.</dd>
+            <dd>NOTE: They have changed their name to Industry IoT Consortium</dd>
+            <dt><a href="https://opcfoundation.org">OPC Foundation</a></dt>
```

Comment:
Can be removed, since we will have a new entry about the planed OPC UA cooperation

Reply:
Done